### ISSUE TITLE
Absolute position

### ISSUE THREAD

**Issue by brandontan903** [beaverbuilder/feature-requests#81]
Some web design I'm going to make need absolute position.

Using Beaver Builder, creating such design is really hard and frustrating. There is no such option currently to do that.
I'm trying to use CSS but it does not looks nice.

Beaver Builder nowadays really need to have absolute position.

I realized many other page builders have this feature.

This is example in other page builder, just to get in idea:
https://www.youtube.com/watch?v=vJ851SD-6u0

**Comment by Messa1** [comment]
I also talk with the support about your wish. I also want a absolute position tool.

**Comment by Messa1** [comment]
Also the https://www.editorx.com/ - has a absolute position tool.

**Comment by hendratok** [comment]
yeah we really need this option today. all other builder have this feature
